README.md

```diff
@@ -134,6 +134,16 @@ dets := classifier.RunCascade(cParams, angle)
 dets = classifier.ClusterDetections(dets, 0.2)
 ```
 
+A note about imports, in order to decode the image you will need to import "image/jpeg" as well as the pigo library. See the following example

```

Comment:
I would require a few small adjustments on the text. Can you modify the the following paragraphs as below:

**A note about imports**: in order to decode the image you will need to import `image/jpeg` or `image/png` (depending on the provided image type) and the Pigo library as well, otherwise you will get a `"Image: Unkown format"` error. See the following example:

```Go
import (
    _ "image/jpeg"
    pigo "github.com/esimov/pigo/core"
)
```

Reply:
Of course, sorry I'm not the greatest but the changes have been made and commits squashed to not pollute the commit history. 

Thanks!